roles/confluent.variables/defaults/main.yml

```diff
@@ -35,11 +35,11 @@ jmxexporter_enabled: false
 jmxexporter_jar_path: /opt/prometheus/jmx_prometheus_javaagent.jar
 
 ### Boolean to have cp-ansible configure components with FIPS security settings
```

Comment:
I think i understand whats going on here and i like it, unfortunately the documentation python script i wrote actually prints out the default... so now no one will know the default is false.... the simple solution is just to put in the annotation. this defaults to false

Reply:
I think I'm actually going to revert this change and go a different route with this. IE force folks to be explicit. All global vars will be treated as fully global. If you want something to be different then whats defined global, you then do the change explicitly to your cluster/resources in question via group/hostvars.